scripts/install-atom.ps1

```diff
@@ -1,4 +1,4 @@
-choco install -y atom
+choco install -y atom --version 0.174.0
```

Comment:
Wow, that's an old version.

Reply:
Do you remember which version you installed last time?